### PR TITLE
core/v1: remove comment about non-existing constants

### PR DESCRIPTION
**Pull request by AkihiroSuda**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Fixed godoc.

`NodeReachable`, `NodeLive`, `NodeSchedulable`, and `NodeRunnable` are mentioned as "built-in set of conditions" but some of them do not exist in the current API.

#### Which issue(s) this PR fixes:

NONE

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -5867,8 +5867,6 @@ const (
 type NodeConditionType string
 
 // These are valid but not exhaustive conditions of node. A cloud provider may set a condition not listed here.
-// The built-in set of conditions are:
-// NodeReachable, NodeLive, NodeReady, NodeSchedulable, NodeRunnable.
```
Comment (liggitt) [inline staging/src/k8s.io/api/core/v1/types.go:5871]: NodeReady and NodeSchedulable still exist and are in use

Comment (bart0sh) [reply staging/src/k8s.io/api/core/v1/types.go:5871]: @AkihiroSuda please address @liggitt's comment, thanks.

Comment (AkihiroSuda) [reply staging/src/k8s.io/api/core/v1/types.go:5871]: `NodeSchedulable` is not defined as a `NodeConditionType`.
It is defined as a string in `pkg/kubelet/events` though.

Comment (AkihiroSuda) [reply staging/src/k8s.io/api/core/v1/types.go:5871]: (And it is obvious that `NodeReady` is defined as a built-in condition here)

Comment (AkihiroSuda) [reply staging/src/k8s.io/api/core/v1/types.go:5871]: Worded as `Relevant events contain "NodeReady", "NodeNotReady", "NodeSchedulable", "NodeNotSchedulable".`.

Also updated `pkgs/apis/core/types.go`.

**Pull request merged**